hphp/runtime/base/runtime-option.cpp

```diff
@@ -1400,6 +1403,9 @@ void RuntimeOption::Load(
     Config::Bind(ServerThreadDropStack, ini, config, "Server.ThreadDropStack");
     Config::Bind(ServerHttpSafeMode, ini, config, "Server.HttpSafeMode");
     Config::Bind(ServerStatCache, ini, config, "Server.StatCache", false);
+    Config::Bind(ServerFastStatCache, ini, config, "Server.FastStatCache", false);
+    Config::Bind(ServerFastStatCacheBucketNum, ini, config, "Server.FastStatCacheBucketNum", 0);
+    Config::Bind(ServerFastStatCacheTTL, ini, config, "Server.FastStatCacheTTL", 5);
```

Comment:
I had changed ServerFastStatCacheTTL to ServerFastStatCacheExpireSeconds in the new commit.